sync/upstream.py

```diff
@@ -721,13 +721,22 @@ def status_changed(git_gecko, git_wpt, sync, context, status, url, sha):
     if status == "pending":
         # Never change anything for pending
         return landed
-
-    if status == "success":
+    previous_status = sync.last_pr_check
+    success, non_success = env.gh_wpt.status_checks_pass(
```

Comment:
I think this whole bit would be easier if it used `get_combined_status` directly (maybe allow `exclude` to be passed into that one too).